presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestDriver.java

```diff
@@ -710,13 +710,18 @@ public void testExecute()
     {
         try (Connection connection = createConnection()) {
             try (Statement statement = connection.createStatement()) {
-                assertTrue(statement.execute("SELECT 123 x, 'foo' y"));
+                assertTrue(statement.execute("SELECT 123 x, 'foo' y, NULL z"));
                 ResultSet rs = statement.getResultSet();
                 assertTrue(rs.next());
                 assertEquals(rs.getLong(1), 123);
                 assertEquals(rs.getLong("x"), 123);
+                assertEquals(rs.wasNull(), false);
                 assertEquals(rs.getString(2), "foo");
                 assertEquals(rs.getString("y"), "foo");
+                assertEquals(rs.wasNull(), false);
+                assertEquals(rs.getLong(3), 0);
```

Comment:
This is probably incorrect since the type of column `z` actually has a unknown type (selecting an untyped literal `NULL` creates a lot of edge cases). We can fix it by casting the `NULL` to a specific type.